### PR TITLE
Adding changes for Module 5 - Docker

### PR DESCRIPTION
**Pull request by rajeshdewan**
Please make sure you are raising this PR against your own repository and not the original. If it says https://github.com/CorndelWithSoftwire in the search bar right now, you are in the wrong place!

Please also don't forget to submit the PR's url to Aptem afterwards (by clicking on the corresponding exercise component, pasting the url in the textbox that appears and then pressing the finish button).

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+ENV FLASK_ENV=production
+ENTRYPOINT ["poetry","run","gunicorn","--bind","0.0.0.0:8000","todo_app.app:create_app()"]
+
+FROM base as development
+
+ENTRYPOINT ["poetry","run","flask","run","--host","0.0.0.0"] 
+
+
+# These are the commands to build the image and run docker
```
Comment (hugh-emerson) [inline Dockerfile:28]: These instructions are in your README (as they should be), so you don't need them here as well. Having two copies of instructions just means one is going to end up out of date.

File: todo_app/data/session_items.py [hunk 2]
```diff
@@ -43,9 +43,27 @@ def doneitems(self):
...
+
+   response = requests.request(
+      "GET",
+      url_for_board,
+      params=query
+   )
+   
+   getBoardresponse = response.text
+   getBoardresponse = json.loads(getBoardresponse)
+   return getBoardresponse[0]["id"]
```
Comment (hugh-emerson) [inline todo_app/data/session_items.py]: Why did you make this change?

This means that you don't control which board gets used when you have multiple boards. Most importantly, it means the e2e test can't make the app use a dedicated test board

File: README.md [hunk 3]
```diff
@@ -86,3 +86,23 @@ You should see output similar to the following:
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
 
+
+Docker commands
+These are the commands to build the image and run docker
+For Dev instance
+docker build --target development --tag todo-app:dev .
```
Comment (hugh-emerson) [inline README.md:93]: This is a markdown file (.md file extension). Single line breaks don't lead to line breaks in the formatted file:

https://github.com/rajeshdewan/-DevOps-Course-Starter/blob/40b44d3076c31498b3587fc5ac4944c157733d27/README.md

**Review by hugh-emerson — APPROVED** [review]
All of the module 5 work looks good, but the unrelated change to session_items.py removes some functionality so you probably want to revert that.

Merge when you're ready 👍

File: todo_app/data/session_items.py (hunk 2 above)
Comment (rajeshdewan) [reply todo_app/data/session_items.py]: Reverting the functionality for this change

**Pull request merged**